### DISCUSSION TITLE
Milvus always exits when I insert data

### DISCUSSION THREAD

**Discussion by CharlesWu123** [milvus-io/milvus#34853] [Q&A and General discussion]
<img width="1603" alt="image" src="https://github.com/user-attachments/assets/043b9f00-a6ad-4231-b84a-3559b3103457">

Milvus always exits when I insert data. What is the reason for this?

**Comment by xiaofan-luan** [comment]
seems that you lost your connection to etcd.

1. please check your etcd log and its response time, make sure etcd is deployed on ssds.
2. what is the cpu usage of your milvus? does it reach 100% by the time data inserted

**Reply by CharlesWu123** [reply]
1. The following is the etcd log and There doesn't seem to be anything unusual. Does etcd have to be deployed on SSD? I am currently deploying it on a regular hard drive
![image](https://github.com/user-attachments/assets/78d15c3a-7613-4e57-92aa-4e8dd3673ed3)

**Reply by xiaofan-luan** [reply]
if you take a look at the latency, it's already hundred milli seconds.
I'm doubting your etcd is running on SSD.

**Reply by CharlesWu123** [reply]
My etcd does not run on ssd, it only runs on regular hard drives.  What should be its normal latency？ I looked at other times and found that the latency was mostly above 100 ms, and the data was being inserted normally. However, at some point, Milvus encountered an issue with exiting.

**Reply by xiaofan-luan** [reply]
milvus exit becuse etcd timeout and session loss.
you can try to change 
common.session.ttl to larger value (See 300 s) instead of 30s. but this will longer the failure recovery time.

**Reply by CharlesWu123** [reply]
Okay, thank you for your help.

**Comment by xiaofan-luan** [comment]
We don't recommend to deploy etcd on HDDs.  This is super unstable and SSD machine is something not difficult to get today. So we highly suggest u to directly use cloud or use a SSD machine if you do want to self host

**Reply by CharlesWu123** [reply]
OK

**Reply by CharlesWu123** [reply]
@xiaofan-luan My data insertion speed is relatively slow, is this also because etcd is deployed on HDD?
